cmd/spiffe-aws-assume-role/cli/cli_test.go

```diff
@@ -0,0 +1,98 @@
+package cli
+
+import (
+	"github.com/aws/aws-sdk-go/service/sts"
+	"github.com/square/spiffe-aws-assume-role/internal/mocks"
+	"github.com/square/spiffe-aws-assume-role/internal/test"
+	"github.com/square/spiffe-aws-assume-role/pkg/credentials"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+	"testing"
+	"time"
+)
+
+var NoContext *CliContext = nil
```

Comment:
Although Goland is good about adding parameter name sugar to constant arguments, this functionality is lacking in most CR tools (including this one). To help with clarity I like to introduce well-named constants so that it's obvious to the reader what the meaning of _nil_, _true_, etc. might be without having to poke around at the invoked method.